### ISSUE TITLE
Feature request: use modern config template

### ISSUE THREAD

**Issue by jef** [linuxserver/docker-rutorrent#148]
[linuxserverurl]: https://linuxserver.io
[![linuxserver.io](https://raw.githubusercontent.com/linuxserver/docker-templates/master/linuxserver.io/img/linuxserver_medium.png)][linuxserverurl]

------------------------------

## Expected Behavior

A more modern configuration that has descriptions on the config file. Something along the lines like this https://github.com/rakshasa/rtorrent/wiki/CONFIG-Template

## Current Behavior

Non descriptive rtorrent config file.

**Comment by piramiday** [comment]
now that #161 got merged and #157 got closed, allow me to comment on this.

it's a great thing to update the `rtorrent.rc` included in the container but, as of now, it's a wasted occasion.
since the `rc` file is being updated, it should be updated to the new format of the configuration as detailed in the official rtorrent wiki, as per the original request of this issue.

among the many things, the new `rc` (merged with #161) is missing the important `data_path` method, which allow for robust user scripts:
```
method.insert = d.get_data_full_path, simple, "branch=((d.is_multi_file)),((cat,(d.directory))),((cat,(d.directory),/,(d.name)))"
```
and, above all, is not exploiting the **awesome drop-in feature** documented here:
https://rtorrent-docs.readthedocs.io/en/latest/use-cases.html#drop-in-config
and intelligently requested here https://github.com/linuxserver/docker-rutorrent/pull/157#issuecomment-627943760 ...

since many new users will start editing their rtorrent configs as per the new guidelines, it would make sense to adhere to those standards as much as possible.
the drop-in config, together with a smart placement of the config folder and an appropriate remapping on the docker host, would let users effortlessly instrument their rtorrent with new instructions without having to modify the included rtorrent script.

**Comment by j0nnymoe** [comment]
We welcome PR's if you feel this would be an improvement to this container.

**Comment by piramiday** [comment]
well, #157 was trying to do that, and it got closed :man_shrugging:

**Comment by j0nnymoe** [comment]
It got closed in favour of another PR.

**Comment by piramiday** [comment]
not quite, it got closed in favour of an inferior PR -- while #161 slightly updated only a few commands, #157 was shooting for a more radical update with several improvements to bring the rc file in line with current guidelines, which would have been even better with the drop-in snippet, as requested.
in other words, they were not equivalent, and #161 did not supersede #157...

**Comment by jef** [comment]
I agree. #161 was a short term updated before updating more config. #157 should really be reopened and at least modified to adhere to #161's changes. I don't believe this can be closed in light of #161's merging.
